packages/ts/react-crud/src/autoform.tsx

```diff
@@ -68,8 +68,30 @@ export type AutoFormProps<M extends AbstractModel = AbstractModel> = ComponentSt
      */
     disabled?: boolean;
     /**
-     * Allows to customize the layout of the form by providing a custom renderer.
-     * Check the component documentation for details.
+     * Allows to customize the layout of the form by providing a custom
+     * renderer. The renderer receives the form instance and the pre-rendered
+     * fields as props. The renderer can either reuse the pre-rendered fields in
+     * the custom layout, or render custom fields and connect them to the form
+     * manually.
+     *
+     * Check the component documentation for details and examples.
+     *
+     * Example using pre-rendered fields:
+     * ```tsx
+     * <AutoForm layoutRenderer={({ children }) =>
```

Comment:
`layoutRenderer={VerticalLayout} should work as well in this case

Reply:
Sure, but I was mostly trying to show how to make use of the prop in case you want to have a more complex layout.

Reply:
What about two examples then? As we're trying to minimize code, I see value in suggesting the shortest one too.

Reply:
Well, I don't think that just rendering the fields in a vertical layout instead of a form layout is a practical use-case. So maybe just add another element somewhere in there?

Reply:
So how about this:
```tsx
<AutoForm layoutRenderer={({ children }) =>
  <VerticalLayout>
    {children}
    <p>All data is collected anonymously.</p>
  </VerticalLayout>
} />
```

Reply:
> Well, I don't think that just rendering the fields in a vertical layout instead of a form layout is a practical use-case.
I still see some value in showing that you can use a layout directly. And I like your last example which makes explicit declaration necessary. I let you decide.